sda-svc/templates/s3-inbox-deploy.yaml

```diff
@@ -126,7 +125,7 @@ spec:
           value: "{{ .Values.global.elixir.oidcdHost }}{{ .Values.global.elixir.jwkPath }}"
         {{- end }}
         - name: SERVER_LOG_LEVEL
```

Comment:
not a comment for the PR but at some point we should standardize these env vars as other services seem to use `LOG_LEVEL`